docs/docs/deployment/deploy-model-locally/index.mdx

```diff
@@ -22,7 +22,7 @@ Before deploying, you must have an MLflow Model. If you don't have one, you can
 Remember to note down the model URI, such as `runs:/<run_id>/<artifact_path>` (or `models:/<model_name>/<model_version>` if you registered the model in the [MLflow Model Registry](/model-registry)).
 
 Once you have the model ready, deploying to a local server is straightforward. Use the <Link to="/api_reference/cli.html#mlflow-models-serve" target="_blank">mlflow models serve</Link> command for a one-step deployment.
-This command starts a local server that listens on the specified port and serves your model.
+This command starts a local server that listens on the specified port and serves your model. Refer to the CLI reference for available options.
```

Comment:
Can we add a link to `the CLI reference`?

Reply:
This is the one just above (/api_reference/cli.html#mlflow-models-serve), so wasn't sure if we want to add the link here again. Anyway, added the link here.